### ISSUE TITLE
date item type: entering numbers via numpad not possible

### ISSUE THREAD

**Issue by selinakreinhard** [rubenarslan/formr.org#323]
Item type "date": 
Answering this item type by entering the number via the numeric keypad is not possible (, but choosing the number using the calendar is). When the first digit is entered, the cursor skips back to the left (where 'day' should be entered). Browser = firefox
Not sure if it is intended or a bug, but during testing some users preferred to enter their date via numpad (they found the calendar confusing).

**Comment by selinakreinhard** [comment]
similarly, a problem occurs with itemtype "month" only if the keyboard is used. If the input of a month item is constrained (e.g. 1.1.1900, now), selection via calendar works fine, but invalid input (e.g., Jan 2019) can be entered via keyboard. The error message occurs only after the page was submitted and all previously given answers on this page are gone.

**Comment by rubenarslan** [comment]
see #334

**Comment by mpw-wwu** [comment]
@rubenarslan, I'd suggest to remove the milestone tag from this, as updating the polyfills will be a hugh task and should not be done in a hotfix.

**Comment by rubenarslan** [comment]
The date input is still replaced by the webshim polyfill, so this rides on #334 and is deferred to form_v2.
